.github/generate-envs.py

```diff
@@ -237,6 +237,26 @@
         "python-version": "3.8",
         "group": "ci",
     },
+    # Test Xcelium on Ubuntu
```

Comment:
```suggestion
    # Test Cadence Xcelium on Ubuntu
```
(you also say "Siemens Questa")